lib/container/RemoteRuntimeModule.js

```diff
@@ -32,7 +32,9 @@ class RemoteRuntimeModule extends RuntimeModule {
 		const chunkToRemotesMapping = {};
 		/** @type {Record<ModuleId, [string, string, string | number | null]>} */
 		const idToExternalAndNameMapping = {};
-		for (const chunk of /** @type {Chunk} */ (this.chunk).getAllAsyncChunks()) {
+		for (const chunk of /** @type {Chunk} */ (
```

Comment:
the same is applied in module federation v2, has been for about a year where this is my default.